csrc/velox/column.cpp

```diff
@@ -445,8 +458,27 @@ std::unique_ptr<OperatorHandle> OperatorHandle::fromUDF(
 
   velox::TypePtr outputType =
       velox::resolveFunction(udfName, inputRowType->children());
+
   if (outputType == nullptr) {
-    throw std::runtime_error("Request for unknown Velox UDF: " + udfName);
+    auto children = inputRowType->children();
+
+    std::string signature;
+    folly::join(", ", children.begin(), children.end(), signature);
+    signature = "(" + signature + ")";
+    folly::toLowerAscii(signature);
+
+    auto allSignatures = velox::getFunctionSignatures();
+    auto it = allSignatures.find(udfName);
+    if (it == allSignatures.end()) {
+      throw std::runtime_error(
```

Comment:
shall we just use `VELOX_USER_FAIL` here? :)

Reply:
tried that a bit and found it's actually pretty verbose (see the example below), feel it's more concise to keep the current version.
```
WARNING: Logging before InitGoogleLogging() is written to STDERR
E20220624 11:25:39.216804 641654 Exceptions.h:68] Line: /Users/bearzx/facebookresearch-torcharrow/csrc/velox/column.cpp:483, Function:fromUDF, Expression: blaaaaaa (bigint, bigint), Source: USER, ErrorCode: INVALID_ARGUMENT
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/bearzx/facebookresearch-torcharrow/torcharrow/functional.py", line 241, in firstx
    return _dispatch("firstx", col, num_to_copy)
  File "/Users/bearzx/facebookresearch-torcharrow/torcharrow/functional.py", line 53, in _dispatch
    return op(*args)
  File "/Users/bearzx/facebookresearch-torcharrow/torcharrow/velox_rt/functional.py", line 39, in dispatch
    result_col = ta.generic_udf_dispatch(op_name, *wrapped_args)
RuntimeError: Exception: VeloxUserError
Error Source: USER
Error Code: INVALID_ARGUMENT
Reason: blaaaaaa (bigint, bigint)
Retriable: False
Function: fromUDF
File: /Users/bearzx/facebookresearch-torcharrow/csrc/velox/column.cpp
Line: 483
Stack trace:
# 0
# 1
# 2
# 3
# 4
# 5
# 6
# 7
# 8
# 9
# 10
# 11
# 12
# 13
# 14
# 15
# 16
# 17
# 18
# 19
# 20
# 21
# 22
# 23
# 24
# 25
# 26
# 27
# 28
# 29
# 30
# 31
# 32
# 33
# 34
# 35
```

Reply:
i think what we need to do is to register the exception to Python so we can catch it in Python, etc. But for now let' use `runtime_error` :)